internal/blocksync/reactor.go

```diff
@@ -174,9 +175,9 @@ func (bcR *Reactor) OnStop() {
 }
 
 // StreamDescriptors implements Reactor.
-func (*Reactor) StreamDescriptors() []p2p.StreamDescriptor {
-	return []p2p.StreamDescriptor{
-		&tcpconn.ChannelDescriptor{
+func (*Reactor) StreamDescriptors() []abstract.StreamDescriptor {
```

Comment:
Can you think of a better name than `abstract`? `common`?

Reply:
Is there a reason for these interfaces not be on the base `p2p` package? Is an import loop issue?

Reply:
Could them be place in `p2p/transport` as well?